### PR TITLE
Added Proxy section for vagrant

### PR DESCRIPTION
**Pull request by jjasghar**
- Added vagrant proxy section, this seems like something that should be documented.

### CODE REVIEW

File: source/docs/getting-started/faq.md [hunk 1]
```diff
@@ -86,3 +86,13 @@ You can edit the `.kitchen.yml` with a `transport` option, for instance:
...
+
+### I need to set up a cache or proxy using Vagrant, what are some options for me?
+
+So there are a few things that already exist that sort of cover this in the kitchen world:
+
+- the system level ENV proxies that can be passed through
+- Example of using [polipo](https://www.irif.univ-paris-diderot.fr/~jch/software/polipo/) locally https://gist.github.com/fnichol/7551540
+- vagrant cachier https://github.com/fgrehm/vagrant-cachier
+
+For the system level proxies and the polipo/squid locally you don't need to do much but you'll still need to edit the conf files for yum/apt as appropriate. Vagrant-cachier requires a custom vagrantfile_erb at this point but we've had lots of interest in automatically enabling it if present or at the very least making it a toggleable option.
```
Comment (smurawski) [inline source/docs/getting-started/faq.md]: Maybe explain what environmental variables we look for?  And how to use the .kitchen.yml config with empty strings to skip pasing them along?

**Comment by mwrock** [comment]
I think we could use a separate section above this just covering proxies in general. It seems at least every week at least one person trips over that. That could be handled in a separate PR.

**Comment by jjasghar** [comment]
Agreed. There's a scoping issue here though. Depends on how detailed we want to get with this. I'm worried we might end up creating a complete "section" for this; instead of just a FAQ.

**Comment by mwrock** [comment]
heh yeah. It actually needs one.

**Pull request merged**